sources/parsley/main.py

```diff
@@ -19,13 +20,16 @@
 class SerialCommunicator:
     def __init__(self, port, baud, timeout):
         self.port = port
-        self.serial = serial.Serial(port, baud, timeout=timeout)
+        # self.serial = serial.Serial(port, baud, timeout=timeout)
+        pass
 
     def read(self):
-        return self.serial.read(4096)
+        # return self.serial.read(4096)
+        return b''
 
     def write(self, msg):
-        self.serial.write(msg)
+        # self.serial.write(msg)
+        pass
```

Comment:
are these commented out code intended

Reply:
I believe  so, should I just remove them?

Reply:
how is parsley going to send and receive serial without these?

Reply:
I think we changed the way parsley sends and receives  serial for testing purposes. If you run individual parsley instances on any specific COM. It should work for testing purposes.

Reply:
For testing yeah, but we probably don't want these testing code merged right?

Reply:
Yeah, I put them back in the recent commit with all the changes